README.md

```diff
@@ -20,7 +20,7 @@ cd ./dev/
 rush build    # Will build all the required packages.
 rush bundle   # Will prepare bundles.
 rush docker:build   # Will build Docker containers for all applications in the local Docker environment.
-docker-compose up -d --force-recreate # Will set up all the containers
+docker compose up -d --force-recreate # Will set up all the containers
```

Comment:
👀

Reply:
Better to switch to new docker compose, https://docs.docker.com/compose/reference/ in all places, not just here and do it in separate PR.

Reply:
Reverted